### PR TITLE
Respect ordering from audienceActions to determine prompts

### PR DESCRIPTION
**Pull request by kristenwang**
b/270965538.

### CODE REVIEW

File: src/runtime/auto-prompt-manager.js [hunk 1]
```diff
@@ -545,20 +526,40 @@ export class AutoPromptManager {
...
+        (action) => action.type === TYPE_CONTRIBUTION
+      );
+
+      if (contributionIndex > 0) {
+        actionToUse = potentialActions[0].type;
+        this.promptDisplayed_ = actionToUse;
+        return actionToUse;
+      }
+
+      // If the first potential action is contribution, and it has never been
```
Comment (ChrisAntaki) [inline src/runtime/auto-prompt-manager.js:539]: What if contribution isn't a potential action? (Ex: contributionIndex was -1)

Comment (kristenwang) [reply src/runtime/auto-prompt-manager.js:539]: When contributionIndex is -1, it will be treated the same as when contributionIndex === 0.

Comment (ChrisAntaki) [reply src/runtime/auto-prompt-manager.js:539]: In that case, should the comment on line 539 change?

Comment (ChrisAntaki) [reply src/runtime/auto-prompt-manager.js]: Thanks!

**Review by ChrisAntaki — APPROVED** [review]

**Pull request merged**